pkg/app/net/unix/accountmanager.go

```diff
@@ -41,3 +83,12 @@ func (i *UserInfo) Username() string {
 	}
 	return i.username
 }
+
+func oauthCallbackURL(host string) string {
+	return "http://" + host + "/oauth2callback"
```

Comment:
Should this be https?

Reply:
The server in the main.go file uses http only (https is added through a reverse proxy when deployed to app engine). So it will be http when running locally. The OAuth spec requires https always except when the host is local (localhost, 127.1 or ::1)